crypto/keyring/keyring.go

```diff
@@ -112,17 +112,15 @@ type Importer interface {
 	// ImportPrivKey imports ASCII armored passphrase-encrypted private keys.
 	ImportPrivKey(uid, armor, passphrase string) error
 
-	// ImportPubKey imports ASCII armored public keys.
-	ImportPubKey(uid string, armor string) error
+	// ImportInfo takes a keyring.Info (in practise, from an old keyring), and
+	// writes it to the current keyring. We use it to migrate Type{Multi,Ledger,Offline}
+	// keyring.Infos.
+	ImportInfo(oldInfo Info) error
 }
 
-// Exporter is implemented by key stores that support export of public and private keys.
+// Exporter is implemented by key stores that support export of private keys.
 type Exporter interface {
-	// Export public key
-	ExportPubKeyArmor(uid string) (string, error)
-	ExportPubKeyArmorByAddress(address sdk.Address) (string, error)
-
-	// ExportPrivKey returns a private key in ASCII armored format.
+	// ExportPrivKeyArmor returns a private key in ASCII armored format.
```

Comment:
This is API breaking, so shouldn't be introduced in this PR if we are backporting it.

Reply:
agree, I'll move this to a seperate PR

Reply:
This should be ok now. Please review again